src/utils/VideoRelevanceManager.ts

```diff
@@ -39,4 +51,12 @@ export class VideoRelevanceManager {
       this.videosToUpdate.clear()
     }
   }
+
+  private async updateLoop(intervalMs: number): Promise<void> {
+    const em = await getEm()
+    while (true) {
+      await this.updateVideoRelevanceValue(em, true)
```

Comment:
if this will force to update al the relevances anyway, do we still need to keep the update for the views tick?

Reply:
I mean, if by chance there would be some ridiculous views peak it would be worth updating it instantly